package/gluon-core/luasrc/usr/lib/lua/gluon/util.lua

```diff
@@ -134,10 +134,14 @@ function get_mesh_devices(uconn)
 	return devices
 end
 
-local function find_phy_by_path(path)
+function find_phy_by_path(path)
```

Comment:
Wrong function: you use *find_phy*, but removed the `local` for *find_phy_by_path*.